src/store/modules/chat/index.js

```diff
@@ -607,6 +596,28 @@ const actions = {
     })
   },
 
+  reInitCommands({ dispatch }, messages) {
+    if (messages && messages.length > 0) {
+      const chats = Object.groupBy(messages, ({ recipientId }) => recipientId)
+      for (const recipientId in chats) {
+        const chatMessages = chats[recipientId]
+        if (Array.isArray(chatMessages)) {
+          const commands = extractCommandsFromMessages(recipientId, chatMessages)
+          if (commands.length > 0) {
+            dispatch(
+              'botCommands/initBotCommands',
+              {
+                partnerId: recipientId,
+                commands
+              },
+              { root: true }
+            )
+          }
+        }
+      }
+    }
+  },
```

Comment:
This logic is more related to `bot-commands` module.
Move the action to `bot-commands/`